__tests__/app.test.js

```diff
@@ -22,133 +20,174 @@ describe('api',()=>{
+            expect(review.hasOwnProperty("owner", expect.any(String))).toBe(
+              true
+            );
+            expect(review.hasOwnProperty("title", expect.any(String))).toBe(
+              true
+            );
+            expect(review.hasOwnProperty("review_id", expect.any(Number))).toBe(
+              true
+            );
+            expect(review.hasOwnProperty("category", expect.any(String))).toBe(
+              true
+            );
+            expect(
+              review.hasOwnProperty("review_img_url", expect.any(String))
+            ).toBe(true);
+            expect(
+              review.hasOwnProperty("created_at", expect.any(Number))
+            ).toBe(true);
+            expect(review.hasOwnProperty("votes", expect.any(Number))).toBe(
+              true
+            );
+            expect(review.hasOwnProperty("designer", expect.any(String))).toBe(
               true
             );
             expect(
-              category.hasOwnProperty("description", expect.any(String))
+              review.hasOwnProperty("comment_count", expect.any(Number))
             ).toBe(true);
           });
-        });
-    });
-  });
-  // Responds with:
 
-  // a review object, which should have the following properties:
+          const foundReview = reviews.find((review) => {
+            if (review.review_id === 2) {
+              return review;
+            }
+          });
 
-  describe("/api/reviews/review_id", () => {
-    it("200 - responds with single review object", () => {
+          expect(foundReview.comment_count).toBe("3");
+        });
+    });
+    it("200 GET - the reviews should be sorted by date in descending order", () => {
       return request(app)
-        .get("/api/reviews/2")
+        .get("/api/reviews")
         .expect(200)
         .then(({ body }) => {
-          const review = body.review;
-          //Should be an Object
-
-          console.log(review);
-
-          expect(review.review_id).toBe(2);
-          expect(review.title).toBe('Jenga');
-          expect(review.category).toBe('dexterity');
-          expect(review.designer).toBe('Leslie Scott');
-          
-          expect(review.owner).toBe('philippaclaire9');
-          expect(review.review_body).toBe('Fiddly fun for all the family');
-          expect(review.review_img_url).toBe('https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700');
-          expect(review.created_at).toBe('2021-01-18T10:01:41.251Z');
-          expect(review.votes).toBe(5);
+          const reviews = body.reviews;
+
+          const reviewDates = reviews.map((review) => {
+            return review.created_at;
+          });
+          expect(reviewDates).toBeSorted({ descending: true });
         });
     });
+  });
 
-    it("400 - responds with msg bad request", () => {
+  describe("/api/reviews/:review_id/comments", () => {
+    it("201 POST - responds with the posted comment", () => {
```

Comment:
We can add another 201 test for if we try to make a post with unnecessary properties - for example 
{username: "mallionaire", body: "This is a new comment!", animal: 'monkey' }

In this case, we want it to succesfully post but ignore the animal part - this should be a 201